<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountManage</class>
 <widget class="QWidget" name="AccountManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XMPP Accounts</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../iconsets.qrc">
    <normaloff>:/iconsets/system/default/account.png</normaloff>:/iconsets/system/default/account.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <widget class="QLabel" name="noAccountLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;No accounts created. To create account please click the Add button.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="listLayout">
     <item>
      <widget class="AccountManageTree" name="lv_accs">
       <property name="indentation">
        <number>0</number>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <attribute name="headerCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Server</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="IconButton" name="pb_add" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" stdset="0">
        <string>&amp;Add</string>
       </property>
       <property name="psiIconName" stdset="0">
        <string>psi/addContact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="IconButton" name="pb_modify" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" stdset="0">
        <string>&amp;Modify</string>
       </property>
       <property name="psiIconName" stdset="0">
        <string>psi/options</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="IconButton" name="pb_remove" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" stdset="0">
        <string>Rem&amp;ove</string>
       </property>
       <property name="psiIconName" stdset="0">
        <string>psi/remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>IconButton</class>
   <extends>QWidget</extends>
   <header>iconbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AccountManageTree</class>
   <extends>QTreeWidget</extends>
   <header>accountmanagedlg.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../iconsets.qrc"/>
 </resources>
 <connections/>
</ui>
